exercises/src/app.jsx

```diff
@@ -35,7 +42,7 @@ export default class App extends React.Component {
                 <NavItem onClick={this.onNav} href="/search">
                   Search
                 </NavItem>
-                <NavItem onClick={this.onNav} href="/exercise/new">
+                <NavItem onClick={this.onNew} href="/exercise/new">
```

Comment:
it looks like onNav is now unused, can we remove it?

Reply:
Good thought, but onNav is used by the "Search" link.  However it could be rewritten to be `onSearch` since now it's only function.  I'll do that.